plutus-metatheory/test/TestNEAT.hs

```diff
@@ -126,11 +126,12 @@ prop_Term tyG tmG = do
   -- turn it into an untyped de Bruijn term
   tmUDB <- withExceptT FVErrorP $ U.deBruijnTerm tmU
   -- reduce the untyped term
-  tmUDB' <- withExceptT (\e -> (Ctrex (CtrexTermEvaluationFail "untyped CEK" tyG tmG))) $ liftEither $ runUAgda tmUDB
+  tmUDB' <- withExceptT (\e -> Ctrex (CtrexTermEvaluationFail "untyped CEK" tyG tmG)) $
+    pure $ either (\_ -> U.Error ()) id $ runUAgda tmUDB
```

Comment:
Hm... I guess it can simply be 

```
    let tmUDB' = either (\_ -> U.Error ()) id $ runUAgda tmUDB
```

then? I didn't realize during out call that we pretend to handle every kind of error here.

Reply:
actually that's true. Done.